README.md

```diff
@@ -1,71 +1,612 @@
-# Hyperbridge SDK
+# Hyperbridge SDK Documentation
 
-![CI](https://github.com/polytope-labs/hyperbridge-sdk/actions/workflows/test-sdk.yml/badge.svg)
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+![CI](https://github.com/polytope-labs/hyperbridge-sdk/actions/workflows/test-sdk.yml/badge.svg) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-Hyperbridge SDK is a comprehensive solution for cross-chain message indexing and retrieval. This monorepo contains two main packages:
+## Overview
 
-- **@hyperbridge/indexer**: A SubQuery-based indexer for tracking cross-chain messages
-- **@hyperbridge/sdk**: A JavaScript/TypeScript SDK for interacting with the indexed data
+The Hyperbridge SDK is a comprehensive developer toolkit that enables secure, trust-free cross-chain interoperability through cryptographic proofs. Built by Polytope Labs, this SDK provides the essential infrastructure for developers to build mission-critical cross-chain applications that leverage Hyperbridge's coprocessor model for expensive cryptographic operations.
 
-## Packages
+Unlike traditional cross-chain solutions that rely on multi-signature committees (which have resulted in over $2 billion in losses), Hyperbridge SDK implements a **crypto-economic coprocessor** that uses consensus proofs to attest to the correctness of computations performed onchain, enabling truly trustless cross-chain communication.
```

Comment:
Hyperbridge sdk does not implement this it's a tool for interacting with Hyperbridge protocol